apps/common/utils/constants.ts

```diff
@@ -1 +1,13 @@
+import {mainnet} from 'wagmi';
+import {arbitrum, fantom, optimism} from '@wagmi/chains';
+import {localhost} from '@yearn-finance/web-lib/utils/wagmi/networks';
+
 export const DEFAULT_SLIPPAGE = 0.5;
+
+export const SUPPORTED_CHAINS = [
+	mainnet,
+	optimism,
+	fantom,
+	arbitrum,
+	localhost
+];
```

Comment:
I would use a map here rather than an array, a bit more clear what we're getting `SUPPORTED_CHAINS.mainnet` rather than `SUPPORTED_CHAINS[0]`
```suggestion
export const SUPPORTED_CHAINS = {
	mainnet,
	optimism,
	fantom,
	arbitrum,
	localhost
};
```